### DISCUSSION TITLE
Can Material for MkDocs support Chinese Now?

### DISCUSSION THREAD

**Discussion by NoneTypeCoder** [squidfunk/mkdocs-material#3025] [Q&A] [answered]
The version of lunr-language has reached 1.9.1, and it supports Chinese. 
Can Material for MkDocs support Chinese?
And I can set yml file like:
 plugins:
 - search:
      lang: zh
Now I set lang as ja， but it dose not work well when searching something.

**Comment by squidfunk** [comment, marked as answer]
See my comment https://github.com/mkdocs/mkdocs/issues/2509#issuecomment-882689383
